public/app/features/alerting/unified/types/rule-form.ts

```diff
@@ -49,6 +49,7 @@ export interface RuleFormValues {
   folder: Folder | undefined;
   evaluateEvery: string;
   evaluateFor: string;
+  keepFiringFor: string;
```

Comment:
Is this field required or optional?

Reply:
is the same as the evaluationFor, is optional but undefined i equivalent to '0s'. 🤔

Reply:
I think we should make it optional

Reply:
done